### ISSUE TITLE
About DEV validation method

### ISSUE THREAD

**Issue by TomSheng21** [thuml/Transfer-Learning-Library#70]
Thanks for your great work.
I use the model in Tranfer-Learning-Library and the method in DEV to search the best learning rate.
When I select the hyperparameters, the validation set separated from the source domain does not participate in the training, but after selecting the hyperparameters, do I need to train with the complete source domain to get the final model?
And the parameters in the experiment in the DEV paper, such as the office31 dataset which has six tasks, the final selected learning rate and loss trade-off are one group or six groups ( a group hyperparameters for each task or the whole dataset)
Looking forward to your answer, thanks a lot.

**Comment by JunguangJiang** [comment]
Maybe you should ask the author of DEV. His email is **youkaichao@gmail.com**.
To have a fair comparison with previous methods, you need to train with the complete source domain to get the final model and the final selected learning rate and loss trade-off are one group.
